scripts/provisioning-terraform.sh

```diff
@@ -238,11 +249,11 @@ fi
 mkdir -p "$(pwd)"/tmp
 
 run_containerized_terraform "${GOOGLE_APPLICATION_CREDENTIALS_PATH}" "${VIAI_CAMERA_INTEGRATION_DIRECTORY_PATH}" version
-run_containerized_terraform "${GOOGLE_APPLICATION_CREDENTIALS_PATH}" "${VIAI_CAMERA_INTEGRATION_DIRECTORY_PATH}" init
+run_containerized_terraform "${GOOGLE_APPLICATION_CREDENTIALS_PATH}" "${VIAI_CAMERA_INTEGRATION_DIRECTORY_PATH}" init -backend-config="bucket=tf-state-${DEFAULT_PROJECT}"
```

Comment:
Check if the backend config file exists first, and throw a meaningful error if it doesn't.

Reply:
This is checked from [line 173](https://github.com/GoogleCloudPlatform/solutions-viai-edge-provisioning-configuration/pull/40/files#diff-f7871aba0c35160d7653778beab0dc5f37e9b1e7bb44e84b8f05dafa95147c53R173)

Reply:
There is a check starting from [line 180](https://github.com/GoogleCloudPlatform/solutions-viai-edge-provisioning-configuration/pull/40/files#diff-f7871aba0c35160d7653778beab0dc5f37e9b1e7bb44e84b8f05dafa95147c53R180)

Reply:
Can you double check those links? I don't see what you mean. Thanks!

Reply:
The newly added function creates the bucket and verify it is created successfully [here](https://github.com/GoogleCloudPlatform/solutions-viai-edge-provisioning-configuration/pull/40/files#diff-b23204e92f793e233cf28605758e6f1d4849830577492c85f00593f977527e27R218)